docs/content/reference/static-configuration/cli.md

```diff
@@ -1,3 +1,8 @@
+---
+title: "Traefik Static Configuration Documentation"
+description: "Learn how to store Traefik Proxy configurations in static files. Read the technical documentation."
```

Comment:
This description sounds weird, because we are describing the static configuration cli flags, not related to static files.